src/molecules/formfields/CreditCardField/Readme.md

```diff
@@ -1,5 +1,27 @@
 CreditCardField Example:
 
 ```example
-    <CreditCardField />
+    <CreditCardField>
+      <CreditCardField.Top>
+        <TextField
+          placeholder='Credit Card Number'
+          noBaseStyle
+        />
+      </CreditCardField.Top>
+
+      <CreditCardField.Bottom>
+        <TextField
+          placeholder='Expiration Date'
+          input={{
+            pattern: '[0-9]{13,16}'
```

Comment:
what is this `pattern` property? I don't see it as a part of redux-form, just curious how we would use this in development